src/components/questionnaire/questionnaire-interaction-screen.tsx

```diff
@@ -0,0 +1,160 @@
+import { KeyboardReturn } from "@mui/icons-material";
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  Checkbox,
+  Divider,
+  FormControlLabel,
+  Typography
+} from "@mui/material";
+import { useState } from "react";
+import type { AnswerOption, Question, Questionnaire } from "src/generated/homeLambdasClient";
+import strings from "src/localization/strings";
+
+interface QuestionnaireInteractionScreenProps {
```

Comment:
Typyically we just call these Props when they are only being used for a specific component. E.g. see new-questionnaire-card.
Also missing TSDOcs